rust/tw_tests/tests/chains/pactus/pactus_sign.rs

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright © 2017 Trust Wallet.
+
+use crate::chains::pactus::test_cases::transfer_1::{
+    pactus_sign_transfer_input, PRIVATE_KEY, SIGNATURE, TRANSACTION_SIGNED_DATA, TX_ID,
+};
+use tw_any_coin::ffi::tw_any_signer::tw_any_signer_sign;
+use tw_coin_entry::error::prelude::*;
+use tw_coin_registry::coin_type::CoinType;
+use tw_encoding::hex::{DecodeHex, ToHex};
+use tw_memory::test_utils::tw_data_helper::TWDataHelper;
+use tw_proto::Pactus::Proto;
+use tw_proto::{deserialize, serialize};
+
+#[test]
+fn test_pactus_sign() {
+    let input = Proto::SigningInput {
+        private_key: PRIVATE_KEY.decode_hex().unwrap().into(),
+        ..pactus_sign_transfer_input()
+    };
+
+    let input_data = TWDataHelper::create(serialize(&input).unwrap());
+
+    let output = TWDataHelper::wrap(unsafe {
+        tw_any_signer_sign(input_data.ptr(), CoinType::Pactus as u32)
+    })
+    .to_vec()
+    .expect("!tw_any_signer_sign returned nullptr");
+
+    let output: Proto::SigningOutput = deserialize(&output).unwrap();
+
+    assert_eq!(output.error, SigningErrorType::OK);
+    assert!(output.error_message.is_empty());
+    assert_eq!(output.transaction_id.to_hex(), TX_ID);
+    assert_eq!(output.signature.to_hex(), SIGNATURE);
+    assert_eq!(
+        output.signed_transaction_data.to_hex(),
+        TRANSACTION_SIGNED_DATA
+    );
```

Comment:
Could you please add a comment like `Successfully broadcasted: https://...`? It will make it easier to find the transaction on explorer

Reply:
Addresses in Testnet start with `tpc1`, unlike Mainnet addresses which start with `pc1`. 
And Ed25519 is not yet active on Mainnet, but it is active on Testnet. 
Can we add this test at a later time?

Reply:
it's ok to add a comment with a testnet transaction link. Did you broadcast the transaction from the test, right? 🙂

Reply:
I tried to do it, but the issue is that the address type needs to support Testnet, which seems unnecessary to me in this case. We will definitely add a test with a link to explorer soon. I hope this won't block us from merging the PR.

Reply:
One of the main criteria is to have a unit test with a successfully broadcasted mainnet transaction. It's ok if the blockchain is still testnet yet to have a testnet transaction, of if the test checks the functionality on testnet only.
When Pactus ed25519 will be on mainnet?

Reply:
> One of the main criteria is to have a unit test with a successfully broadcasted mainnet transaction. It's ok if the blockchain is still testnet yet to have a testnet transaction, of if the test checks the functionality on testnet only. But in this case, there is only one transaction example that never has been broadcasted. When Pactus ed25519 will be on mainnet?

Block 2,320,000 = https://pacviewer.com/block/2320000

Reply:
It is scheduled to get active at first of November. Once it activated, we will add a real test here.

Reply:
Hi @b00f, let's wait for the real test case with a link to the successfully broadcasted transaction.
Please note everything that WalletCore guarantees that everything in master is ready by everyone. If there are no real tests, I can't merge it.